app/views/people/tasks.html.erb

```diff
@@ -0,0 +1,44 @@
+<div class="people_tasks">
+ <h1><%= @person.name.capitalize %>'s Task Details:</h1>
+
+  <div class="profile_top">
+   <h4>To Be Completed:</h4>
+    <p><%= @person.tasks.where(completed_at: nil).count %>/<%= @person.tasks.count %></p>
+   </div>
+
+  <table>
+   <thead>
+   	<th>Tasks:</th>
+   	<th></th>
+   	<th></th>
+   	<th></th>
+   </thead>
+
+   <tbody>
+    <% @person.tasks.each do |task| %>
+			<tr>
+			  <td class="check">
+	  		  <% if task.completed_at != nil %>	
+	  		  	<%= link_to "\u2713", "/mark_incomplete/#{task.id}", method: :patch, class: "incomplete_task" %>
+	  		  <% else %>
+	  		  	<%= link_to "\u25A1", "/mark_complete/#{task.id}", method: :patch, class: "completed_task" %>
+	  		  <% end %>
+			  </td>
+			  <td class="name">
+					<% if task.completed_at != nil %>
+						<span class="completed"><%= task.name %></span>
```

Comment:
It might make sense to include the conditional here to determine which class to display, but then only have one line which contains the span HTML. That way - if anything changed with the span, you would only need to change it in one place.